tests/engine/test_memory_leaks.py

```diff
@@ -65,7 +65,6 @@ def test_leak_local_calcjob(aiida_code_installed):
 
 
 @pytest.mark.skipif(sys.version_info >= (3, 12), reason='Garbage collecting hangs on Python 3.12')
-@pytest.mark.nightly
```

Comment:
btw: maybe you already know, we've moved this test to nightly in #6521. Why do you want to move it back?

Reply:
Because it test the memory leak for SSH and I assume the reason to move it to nightly because it runs very long. It is slow because it use the default safe_interval 30s (which requires 120s in total for four SSH operations.). 
When I run tests locally for https://github.com/aiidateam/aiida-core/pull/6675, I am actually expect it is a normal unit test.

Reply:
How long does it take now?

Reply:
It takes 1*4 + (~1) seconds to finish. Depends on how much I set for the safe_interval.